interface-spec.md

```diff
@@ -18,17 +18,19 @@
 
 **Properties:**
 
-- `String .value` is refined by each interface which extends Node
+- None 
 
 **Methods:**
 
-- `.equals(Node other)` returns true if and only if the argument is a) of the same type b) has the same contents (value and, if applicable, type or language)
+- `.sameTerm(Node other)` returns true if and only if the argument is a) of the same type b) has the same contents (value and, if applicable, type or language)
 
-### IRI extends Node
+### Symbol extends Node
+
+A node named with a URI.  The node is not a URI itsef.
```

Comment:
Discussed in the 2015-11-27 meeting: RDF 1.1 seems to assume that the node _is_ the URI.